### ISSUE TITLE
ActionView::Template::Error (Error: Cannot find module "components/LoginForm")

### ISSUE THREAD

**Issue by chrisdobler** [spark-solutions/spark-starter-kit#57]
Hey guys, I had the app working fine on Heroku before, but just the other day when I deploy I now get this error. I've tried rewinding the project back to known working commits but am still getting the error. I'm wondering if some upgrade to Heroku is breaking the project? I don't know what else could explain it.

**Comment by chrisdobler** [comment]
Okay so I don't know why I wasn't getting this error before, but after I reset my local db and reseeded I started getting this error:

16:44:20 rails-spec.1          | ERROR in ./js/components/LoginForm.js
16:44:20 rails-spec.1          | Module build failed: SyntaxError: /Users/chris/Projects/jollyoyster/jollyoyster_app/client/js/components/LoginForm.js: Only one default export allowed per module.
16:44:20 rails-spec.1          |   44 | }
16:44:20 rails-spec.1          |   45 |
16:44:20 rails-spec.1          | > 46 | export default reduxForm({
16:44:20 rails-spec.1          |      | ^
16:44:20 rails-spec.1          |   47 |   form: 'loginForm',
16:44:20 rails-spec.1          |   48 |   fields
16:44:20 rails-spec.1          |   49 | })(LoginForm)

I made a pull request for the changes that I made.  I don't know enough about redux to know if this is the correct change to make, but it works now for me and addressed several other issues I was having as well.

https://github.com/spark-solutions/spark-starter-kit/pull/58
